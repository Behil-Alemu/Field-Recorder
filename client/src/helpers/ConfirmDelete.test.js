import React from 'react';
import { render } from '@testing-library/react';
import ConfirmDelete from './ConfirmDelete';

it("renders without crashing", function() {
  render(<ConfirmDelete />);
});

it('matches snapshot for danger', function() {
    const { asFragment } = render(<ConfirmDelete />);
	expect(asFragment()).toMatchSnapshot();
});


