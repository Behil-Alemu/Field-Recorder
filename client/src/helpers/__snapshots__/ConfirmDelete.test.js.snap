// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot for danger 1`] = `<DocumentFragment />`;
